charts/cass-operator/values.yaml

```diff
@@ -0,0 +1,66 @@
+# Default values for cass-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# NOTE I have some TODO questions for reviewers, some of which are about
+# auto generated properties.
+
+replicaCount: 1
+
+image:
+  repository: docker.io/datastax/cass-operator
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: master-a45fd80b40e0129e8883a953d595ca85e4b7ce79
+  # TODO It better to keep this commented out or set it to empty string?
+  # registryOverride:
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+
+serviceAccount:
+  # TODO Should we have a create property? Don't we always want to create a service account?
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+# TODO Do we need this for an operator deployment?
+podAnnotations: {}
+
+# TODO Do we need this for an operator deployment?
+podSecurityContext: {}
```

Comment:
I'll go with the same opinion as above, I think one securityContext per deployment of k8ssandra should be enough. I'm struggling to think why some parts would have different securityContexts.

Reply:
>I'm struggling to think why some parts would have different securityContexts.

In terms of security best practices I think it make a lot more sense for each application to run owner its own distinct user. I don't think it is a good idea for example for the Cassandra user to be used for every container created and run in k8ssandra.